src/main/java/uk/gov/digital/ho/hocs/casework/api/StageResource.java

```diff
@@ -62,7 +62,15 @@ ResponseEntity<CreateStageResponse> createStage(@PathVariable UUID caseUUID, @Re
     @Authorised(accessLevel = AccessLevel.READ)
     @PutMapping(value = "/case/{caseUUID}/stage/{stageUUID}/recreate")
     ResponseEntity recreateStageTeam(@PathVariable UUID caseUUID, @PathVariable UUID stageUUID, @RequestBody RecreateStageRequest request) {
-        stageService.recreateStage(caseUUID, request);
+        stageService.createStage(caseUUID, new CreateStageRequest(
```

Comment:
Will this go through before the other change that removes this? This is now `@Deprecated` so I don't feel it needs changing as nothing will end up through this path.

Reply:
it will if the BPMN commands it so? no?

Reply:
This method isn't called anymore in workflow and the create endpoint is used in its lieu. This is only here for overhang at the minute and should be removed on the next `Major` change. 

This was only ever called by the `bpmnService` in workflow and was conditional.